src/components/camera/client.cpp

```diff
@@ -0,0 +1,77 @@
+#include <algorithm>
+#include <common/utils.hpp>
+#include <components/camera/camera.hpp>
+#include <components/camera/client.hpp>
+#include <config/resource.hpp>
+#include <string>
+#include <utility>
+
+#include "common/v1/common.pb.h"
+#include "component/camera/v1/camera.grpc.pb.h"
+
+std::string normalize_mime_type(const std::string& str);
```

Comment:
Can we just define `normalize_mime_type` up here so we don't have to forward declare it?